src/redux/types/actions/motion.ts

```diff
@@ -134,7 +134,11 @@ export type MotionActionTypes =
         userAddress: Address;
         colonyAddress: Address;
         extensionAddress: Address;
-        motionIds: string[];
+        motionIds: {
+          databaseMotionId: string;
+          colonyAddress: string;
+          extensionAddress: string;
+        }[];
```

Comment:
I wonder if there is a better name than `motionIds` now that this is an object

Reply:
Good catch, @iamsamgibbs ! thanks!